internal/pkg/cli/deploy/static_site.go

```diff
@@ -143,3 +148,20 @@ func (d *staticSiteDeployer) stackConfiguration(in *StackRuntimeConfiguration) (
 	}
 	return conf, nil
 }
+
+func (d *staticSiteDeployer) validateAlias() error {
+	if d.staticSiteMft.Alias == "" {
+		return nil
+	}
+
+	hasImportedCerts := len(d.envConfig.HTTPConfig.Public.Certificates) != 0
+	if hasImportedCerts {
+		return fmt.Errorf("cannot specify alias when env %q imports one or more certificates", d.env.Name)
```

Comment:
:wut: so the imported certificates on ELB limits if you can use `alias` for a static site?

Maybe I'm miss understanding, but my expectation would be that users can specify both `alias` and `certs` in a `Static Site` manifest

Reply:
you're 100% right that it doesn't make sense - i was just thinking to start off with the same restrictions we have for NLB until we can validate (and fix, if necessary) that the cert lambda will work with imported certificates.